File: Curso_Dart/operadores/bin/operadores.dart
import 'package:operadores/operadores.dart' as operadores;

void main(List<String> arguments) {
  // Operadores aritméticos
  int suma = 8 + 9;
  int resta = 15 - 3;
  int multiplicacion = 3 * 6;
  double division = 25 / 3;

  print(suma);
  print(resta);
  print(multiplicacion);
  print(division);

  // Operadores relacionales y de igualdad
  print(2 != 2);
  print(1 == 2);
  print(2 > 3);
  print(2 < 3);
  print(2 <= 3);
  print(2 >= 3);

  // Operadores lógicos
  // AND
  print(2 == 2 && 2 > 3);
  // OR
  print(2 == 2 || 2 > 3);
}

File: Curso_JAVA/Fundamentos/src/com/curso/operadores/Operadores.java
package com.curso.operadores;

public class Operadores {

    public static void main(String[] args) {
	    // write your code here
        int a = 5;
        int b = 3;
        int resultado;

        resultado = a + b;
        System.out.println("resultado suma = " + resultado);

        resultado = a - b;
        System.out.println("resultado resta = " + resultado);

        resultado = a * b;
        System.out.println("resultado multiplicacion = " + resultado);

        resultado = a / b;
        System.out.println("resultado division = " + resultado);

        resultado = a % b;
        System.out.println("resultado módulo = " + resultado);

        if (a % 2 == 0)
        {
            System.out.println("Es número par");
        }
        else
        {
            System.out.println("Es número impar");
        }
    }
}
